Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition = "2018"
 [dependencies]
 bitflags = "1.0"
 getopts = "0.2.21"
+getter_rules = { default-features = false, package = "fix-getters-rules", git = "https://github.com/fengalin/fix-getters", tag = "0.1.0" }
```

Comment:
This should probably get a better name so it can go on crates.io without looking strange :) Any suggestions?

Reply:
Not a blocker for merging this now though, just before the release.

Reply:
The actual crate name is `fix-getters-rules`, is that better?

Reply:
Yeah